src/mtype.d

```diff
@@ -8564,39 +8551,24 @@ public:
         Dsymbol s;
         static if (LOGDOTEXP)
         {
-            printf("TypeClass::dotExp(e='%s', ident='%s')\n", e.toChars(), ident.toChars());
-        }
-        if (e.op == TOKdot)
-        {
-            DotExp de = cast(DotExp)e;
-            if (de.e1.op == TOKscope)
-            {
-                ScopeExp se = cast(ScopeExp)de.e1;
-                s = se.sds.search(e.loc, ident);
-                e = de.e1;
-                goto L1;
-            }
+            printf("TypeClass::dotExp(e = '%s', ident = '%s')\n", e.toChars(), ident.toChars());
```

Comment:
When I comapre the two functions code by using diff tool, here had had a small difference. Adding spaces around operator `=` is normal code style in dmd code base, so I synced the debug message with:
```d
printf("TypeStruct::dotExp(e = '%s', ident = '%s')\n", e.toChars(), ident.toChars());
```